services/motion/builtin/builtin_test.go

```diff
@@ -508,7 +508,6 @@ func TestMoveOnMapLongDistance(t *testing.T) {
 }
 
 func TestMoveOnMapPlans(t *testing.T) {
-	t.Parallel()
```

Comment:
is removing these needed to fix the flakey test?

Reply:
I couldnt replicate the failure on my machine so I think it was happening because the CI machine was in contention.  I'm hoping that removing these will force it to run correctly as contention will be lower